tools/tests.py

```diff
@@ -187,12 +189,6 @@
         "dependencies": [MBED_LIBRARIES, TEST_MBED_LIB],
         "automated": True,
         "peripherals": ["analog_loop"],
-        "mcu": ["LPC1768", "LPC2368", "LPC2460", "KL25Z", "K64F", "K66F", "K22F", "LPC4088", "LPC1549",
```

Comment:
I'm thinking we probably don't want to remove the non-ST boards from this list, as the other boards still require some customization of the pins. Would you mind adding the non-ST boards back to this list?